### ISSUE TITLE
[Feature Request] Ability to list and remove, non-existent files' rules

### ISSUE THREAD

**Issue by phpcitizen** [evilsocket/opensnitch#920]
Being able to list non existent executable in rules, so the rule can be easily removed.

**Comment by gustavo-iniguez-goya** [comment]
hi @phpcitizen ,

by non-existent files' rules do you mean temporary rules? put an example please so I can undesrtand it better.

**Comment by phpcitizen** [comment]
Sure, I mean files removed from the system do not need firewall rules anymore, so we can detect and remove them. Like a button with "find invalid rules", then it would list the rules that have no resolved links to an executable anymore.

**Comment by phpcitizen** [comment]
If I may add another feature request.

There should be another default Duration/Action selection option in settings for unattended (when not in front of computer) when the timeout passes of a pop-up.

**Comment by NRGLine4Sec** [comment]
> Sure, I mean files removed from the system do not need firewall rules anymore, so we can detect and remove them. Like a button with "find invalid rules", then it would list the rules that have no resolved links to an executable anymore.

It's not a bad idea. But for some rules, it can be quite complicated.
For example, rules that concern AppImages may be included in the list whose binaries do not exist, while the situation may be quite normal if the AppImage software is not running.

**Comment by phpcitizen** [comment]
Yes, I understand, but I think this can be mitigated with pinning some rules...
